idea_town/frontend/static-src/app/views/experiment-page.js

```diff
@@ -185,32 +185,32 @@ export default PageView.extend({
     evt.target.style.width = width + 'px';
     evt.target.classList.add('state-change');
 
-    this.updateAddon(true, this.model);
-
-    app.on('webChannel:addon-install:install-ended', () => {
-      this.model.enabled = !this.model.enabled;
+    // TODO: Should this maybe be an event listener at a global level?
```

Comment:
About that TODO: I think it might make sense to abstract these install/uninstall listeners out into an Addon model, and toggle view changes by listening to `change` events on that model's attributes, something like `addonModel.on('change:isInstalled', cb)` or whatever you want to call the attrs. Dunno if you want to fold that into this PR or file a followup bug.